port/linux/ipadapter.c

```diff
@@ -1344,11 +1339,6 @@ connectivity_ipv4_init(ip_context_t *dev)
     OC_ERR("setting pktinfo IPv4 option %d\n", errno);
     return -1;
   }
-  if (setsockopt(dev->mcast4_sock, SOL_SOCKET, SO_REUSEADDR, &on, sizeof(on)) ==
```

Comment:
are you sure that the multicast socket is not fixed?

Reply:
No. this a bug - multicast socket must use `SO_REUSEADDR`. - Fixed